ui_advanced_config.py

```diff
@@ -25,116 +28,82 @@ def __init__(self, master=None, logger=None):
-        self.componentsTypeAttr5Label = uiLabel(master=self.componentsColumn, text="")
-        self.componentsTypeAttr5Entry = uiEntry(master=self.componentsColumn)
-        self.componentsTypeAttr6Label = uiLabel(master=self.componentsColumn, text="")
-        self.componentsTypeAttr6Entry = uiEntry(master=self.componentsColumn)
-        self.componentsTypeAttr7Label = uiLabel(master=self.componentsColumn, text="")
-        self.componentsTypeAttr7Entry = uiEntry(master=self.componentsColumn)
-
-        self.selectObjectsCombo = uiComboBox(master=self.objectsColumn)
-        self.objectsLabel = uiLabel(master=self.objectsColumn, text="Objects")
-        self.objectsUpdateButton = uiButton(
-            master=self.objectsColumn, command="", text="Update"
-        )
-        self.objectsIDLabel = uiLabel(master=self.objectsColumn, text="ID:")
-        self.objectsIDEntry = uiEntry(master=self.objectsColumn)
-        self.objectsNameLabel = uiLabel(master=self.objectsColumn, text="Name:")
-        self.objectsNameEntry = uiEntry(master=self.objectsColumn)
-        self.objectsFillAliveLabel = uiLabel(
-            master=self.objectsColumn, text="Fill Alive:"
-        )
-        self.objectsFillAliveEntry = uiEntry(
-            master=self.objectsColumn,
-        )
-        self.objectsFillDeadLabel = uiLabel(
-            master=self.objectsColumn, text="Fill Dead:"
+        self.agentFrame = uiQuietFrame(
+            master=self.teamsColumn, borderwidth=5, relief="ridge", sticky="nsew"
         )
-        self.objectsFillDeadEntry = uiEntry(master=self.objectsColumn)
-        self.objectsTextLabel = uiLabel(master=self.objectsColumn, text="Text:")
-        self.objectsTextEntry = uiEntry(master=self.objectsColumn)
-        self.objectsHealthLabel = uiLabel(master=self.objectsColumn, text="Health:")
-        self.objectsHealthEntry = uiEntry(master=self.objectsColumn)
-        self.objectsDensityLabel = uiLabel(master=self.objectsColumn, text="Density:")
-        self.objectsDensityEntry = uiEntry(master=self.objectsColumn)
-        self.objectsCompIDsLabel = uiLabel(master=self.objectsColumn, text="Comp IDs:")
-        self.objectsCompIDsEntry = uiEntry(master=self.objectsColumn)
-        self.objectsPointsCountLabel = uiLabel(
-            master=self.objectsColumn, text="Points Count:"
+        self.callsignLabel = uiLabel(master=self.agentFrame, text="Callsign:")
+        self.callsignEntry = uiEntry(master=self.agentFrame)
+        self.squadLabel = uiLabel(master=self.agentFrame, text="Squad:")
+        self.squadEntry = uiEntry(master=self.agentFrame)
+        self.agentObjectLabel = uiLabel(master=self.agentFrame, text="Object:")
+        self.agentObjectEntry = uiEntry(master=self.agentFrame)
+        self.aiFileLabel = uiLabel(master=self.agentFrame, text="AI File:")
+        self.aiFileEntry = uiEntry(master=self.agentFrame)
+        self.teamsUpdateButton = uiButton(
```

Comment:
Clicking update button without updating anything seems to delete many values contained within the team object in question. This doesn't seem to replicate for components or objects, so hopeful this is just an accidental oversight somewhere. Picture attached.

![image](https://github.com/Leading-AI-Development-Solutions/MAIA/assets/97774046/2f7ba2d2-de0e-4a30-978e-f0342c490277)

Reply:
Once again, I think this is because there are two teamAlpha values. I am looking into this now